components/eamxx/src/physics/mam/eamxx_mam_aci_functions.hpp

```diff
@@ -25,11 +25,11 @@ void compute_w0_and_rho(haero::ThreadTeamPolicy team_policy,
         static constexpr auto gravit = C::gravit;  // Gravity [m/s2]
         // Gas constant for dry air [J/(kg*K) or J/Kg/K]
         static constexpr auto rair = C::Rair;
-        Kokkos::parallel_for(
-            Kokkos::TeamVectorRange(team, 0u, top_lev), [&](int kk) {
-              w0(icol, kk)  = 0;
-              rho(icol, kk) = -999.0;
-            });
+        Kokkos::parallel_for(Kokkos::TeamVectorRange(team, 0u, top_lev),
```

Comment:
It's interesting that the formatter removes the linebreak here (after `parallel_for(`), but doesn't do the same in this same file, below...